ocf_data_sampler/select/select_spatial_slice.py

```diff
@@ -124,8 +124,10 @@ def _get_idx_of_pixel_closest_to_poi_geostationary(
     center_geostationary = Location(x=x, y=y, coordinate_system="geostationary")
 
     # Check that the requested point lies within the data
-    assert da[x_dim].min() < x < da[x_dim].max()
-    assert da[y_dim].min() < y < da[y_dim].max()
+    assert da[x_dim].min() < x < da[x_dim].max(), \
+        f"{x} not in {da[x_dim].min().values}: {da[x_dim].max().values}"
+    assert da[y_dim].min() < y < da[y_dim].max(), \
+        f"{y} not in {da[y_dim].min().values}: {da[y_dim].max().values}"
```

Comment:
sorry to nitpick, but maybe this could say something like not inside interval/is outside coordinates? Took me a sec to figure out what "x not in {number} : {number}" meant

Reply:
not its good to nitpick, yea, sure Ill do that